x-pack/plugins/security_solution/public/common/hooks/use_experimental_features.ts

```diff
@@ -14,14 +14,16 @@ import {
 
 const allowedExperimentalValues = getExperimentalAllowedValues();
 
-export const useIsExperimentalFeatureEnabled = (feature: keyof ExperimentalFeatures): boolean =>
-  useSelector(({ app: { enableExperimental } }: State) => {
-    if (!enableExperimental || !(feature in enableExperimental)) {
-      throw new Error(
-        `Invalid enable value ${feature}. Allowed values are: ${allowedExperimentalValues.join(
-          ', '
-        )}`
-      );
-    }
-    return enableExperimental[feature];
-  });
+export const useIsExperimentalFeatureEnabled = (feature: keyof ExperimentalFeatures): boolean => {
```

Comment:
I copied this implementation from Sergi's branch so we don't get conflicts when he merges it.